src/web100srv.c

```diff
@@ -1121,10 +1112,10 @@ int run_test(tcp_stat_agent* agent, int ctlsockfd, TestOptions* testopt,
   RTOidle = calc_RTOIdle(vars.Timeouts, vars.CurrentRTO, timesec);
 
   // get timeout, retransmission, acks and dup acks ratios.
-  tmoutsratio = (double) vars.Timeouts / vars.PktsOut;
+  /*tmoutsratio = (double) vars.Timeouts / vars.PktsOut;
   rtranratio = (double) vars.PktsRetrans / vars.PktsOut;
   acksratio = (double) vars.AckPktsIn / vars.PktsOut;
-  dackratio = (double) vars.DupAcksIn / (double) vars.AckPktsIn;
+  dackratio = (double) vars.DupAcksIn / (double) vars.AckPktsIn;*/
```

Comment:
Is this add/revert confusion? Why not just delete these lines instead of commenting them out?